src/unix/system_structs.h

```diff
@@ -168,6 +168,14 @@ typedef struct iovec {
 
 typedef int clockid_t;
 
+#define CLOCK_REALTIME              0
+#define CLOCK_MONOTONIC             1
+#define CLOCK_PROCESS_CPUTIME_ID    2
```

Comment:
Can this be implemented using the time accounting you added for times(2)?

Reply:
Yes, it can, good point. I added support for this clock type too.